src/Compilers/CSharp/Portable/Symbols/ExtraAnnotations.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE;
+using Microsoft.CodeAnalysis.PooledObjects;
+using static Microsoft.CodeAnalysis.CSharp.Symbols.AttributeAnnotations;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    [Flags]
+    internal enum AttributeAnnotations
+    {
+        None = 0,
+        NotNullWhenFalse,
```

Comment:
>NotNullWhenFalse [](start = 8, length = 16)

Please use explicit values for ````[Flags]``` enums

Reply:
Done in follow-up PR https://github.com/dotnet/roslyn/pull/26828